pkg/miniogw/config.go

```diff
@@ -35,6 +35,14 @@ type RSConfig struct {
 	MaxThreshold     int `help:"the largest amount of pieces to encode to. n." default:"50"`
 }
 
+// EncryptionConfig is a configuration struct that keeps details about
+// encrypting segments
+type EncryptionConfig struct {
+	EncKey       string `help:"root key for encrypting the data" default:"supersecretpassword"`
+	EncBlockSize int    `help:"size (in bytes) of encrypted blocks" default:"1024"`
+	EncType      int    `help:"Type of encryption to use" default:"1"`
```

Comment:
Should we add some info about the available encryption types? i.e. `type of encryption to use (1=AES-GCM, 2=SecretBox)`.
I intentionally skip `0=none` to not encouraging it.